todo_app/templates/index.html

```diff
@@ -12,7 +12,21 @@ <h1 class="display-4">To-Do App</h1>
       <h2>Items</h2>
       <ul class="list-group mb-4">
         <!-- List the todo items here -->
+        {% for todoitem in todoitems %}
+        <li>{{ todoitem.title }} </li>
+        {% endfor %}
       </ul>
     </div>
   </div>
+
+  <form action="http://127.0.0.1:5000/add" method="get" class="form-example">
```

Comment:
Similarly here, by specifying the url explicitly we tie this form heavily to our development environment. Instead, we can just specify "/add" here